Chrysalis-Papilio/Soen490-Papilio-Website
# PAPILIO-114-Home Page Completed

## General Information


**Issue number**: Papilio-114


**Task description:**
Home page for the website was completed with missing information (i.e., added an "About Papilio" section and a footer).

## Manual Testing
To test the validation follow these steps:
1. Start the website
`npm start`
2. Completed Home Page should be visible as soon as the website opens


## Testing 
To run all unit tests, run the following code: `npm test`
To run the coverage, run the following code: `npm run coverage`
